src/Package/Util/Dumper.php

```diff
@@ -53,9 +54,10 @@ public function dump(Interfaces\Package $package, $with_version = true)
 
         if ($with_version) {
             $data['version'] = $package->getPrettyVersion();
+            $version_stability = VersionParser::parseStability($data['version']);
             $stability = $package->getStability();
 
-            if ('stable' !== $stability) {
+            if ('stable' !== $stability && 'RC' !== $version_stability) {
```

Comment:
This doesn't look right to me, it won't work for `beta` either, I don't think this code should be here at all, I've raised #202 to discuss that